src/main/java/me/ryanhamshire/GriefPrevention/DatabaseDataStore.java

```
@@ -223,7 +227,9 @@ void initialize() throws Exception
                         }
                     }
                     //otherwise leave it as-is. no harm done - it won't be requested by name, and this update only happens once.
-                    catch (Exception ex) { }
+                    catch (Exception ex)
+                    {
+                    }
```

having curly braces on sameline is preferred _if_ it's blank/swallowing the exception.